src/qt/combobox.cpp

```diff
@@ -20,7 +20,9 @@ class wxQtComboBox : public wxQtEventSignalHandler< QComboBox, wxComboBox >
 {
 public:
     wxQtComboBox( wxWindow *parent, wxComboBox *handler );
-
+    virtual void showPopup();
```

Comment:
Shouldn't we have `wxOVERRIDE` here?

Reply:
@vadz ,
According to the documentation: https://docs.wxwidgets.org/3.1/group__group__funcmacro__misc.html#gac105e0280dbaa177535a20d969e401db

"This macro is useful for writing code which may be compiled by both C++11 and non-C++11 compilers and still allow the use of override for the former."

However wxQt, can be compiled with C++11 only.

So will there be any gain to using wxOVERRIDE?

Reply:
You misunderstood the documentation. The macro is useless when targeting C++11 or later only because in this case you can directly use `override` instead.

In wxQt case we use `wxOVERRIDE` everywhere else, so it would be better to use a macro here too for consistency. If wxQt requires C++11 anyhow, then consider replacing all the occurrences of this macro with `override` in wxQt sources.

Reply:
wxQt does not require C++11.
If it had, it would be automatically be used in the build files. Is it? Please point to at least one line of wx code that requires C++11.

As stated in the wiki, _wxQT needs Qt 5 or later (recommended version is Qt 5.2.1)_, while Qt requires C++11 starting with version 5.7.0. So the requirement comes from Qt, but only for the mentioned version and up.

Reply:
@catalinr ,
Did you try it?
If you try to build wxQt without c++11 support compilation will fail with the message:

"Compilation of wxQt requires C++11 support".

Unfortunately I don't remember which file will require it, but I can try to rebuild it without it and post an error here.